src/pgen/cluster/agn_triggering.cpp

```diff
@@ -234,25 +253,27 @@ void AGNTriggering::ReduceBondiTriggeringQuantities(
       KOKKOS_LAMBDA(const int &b, const int &k, const int &j, const int &i,
                     Real &ltotal_mass_red, Real &lmass_weighted_density_red,
                     Real &lmass_weighted_velocity_red, Real &lmass_weighted_cs_red) {
-        auto &prim = prim_pack(b);
-        const auto &coords = prim_pack.GetCoords(b);
-        const parthenon::Real r2 =
+        if (k >= kb.s && k <= kb.e && j >= jb.s && j <= jb.e && i >= ib.s && i <= ib.e) {                
+          auto &prim = prim_pack(b);
+          const auto &coords = prim_pack.GetCoords(b);
+          const parthenon::Real r2 =
             pow(coords.Xc<1>(i), 2) + pow(coords.Xc<2>(j), 2) + pow(coords.Xc<3>(k), 2);
-        if (r2 < accretion_radius2) {
-          const Real cell_mass = prim(IDN, k, j, i) * coords.CellVolume(k, j, i);
+          if (r2 < accretion_radius2) {
+            const Real cell_mass = prim(IDN, k, j, i) * coords.CellVolume(k, j, i);
 
-          const Real cell_mass_weighted_density = cell_mass * prim(IDN, k, j, i);
-          const Real cell_mass_weighted_velocity =
+            const Real cell_mass_weighted_density = cell_mass * prim(IDN, k, j, i);
+            const Real cell_mass_weighted_velocity =
               cell_mass * sqrt(pow(prim(IV1, k, j, i), 2) + pow(prim(IV2, k, j, i), 2) +
                                pow(prim(IV3, k, j, i), 2));
-          const Real cell_mass_weighted_cs =
+            const Real cell_mass_weighted_cs =
               cell_mass * sqrt(gamma * prim(IPR, k, j, i) / prim(IDN, k, j, i));
 
-          ltotal_mass_red += cell_mass;
-          lmass_weighted_density_red += cell_mass_weighted_density;
-          lmass_weighted_velocity_red += cell_mass_weighted_velocity;
-          lmass_weighted_cs_red += cell_mass_weighted_cs;
-        }
+            ltotal_mass_red += cell_mass;
+            lmass_weighted_density_red += cell_mass_weighted_density;
+            lmass_weighted_velocity_red += cell_mass_weighted_velocity;
+            lmass_weighted_cs_red += cell_mass_weighted_cs;
+          }
+        }  
       },
```

Comment:
What's going on with these changes here?

As far as I can tell the `if k>=kb.s` is new, but it does't change anything as the indices only go over the interior indices anyway.

Reply:
I think this ia a bug because I didn't change this part of the code